driver-core/src/test/unit/com/mongodb/internal/async/client/AsyncAggregateIterableSpecification.groovy

```diff
@@ -147,7 +147,7 @@ class AsyncAggregateIterableSpecification extends Specification {
         operation.getBatchSize() == 99
         operation.getMaxAwaitTime(MILLISECONDS) == 0
         operation.getMaxTime(MILLISECONDS) == 0
-        operation.isAllowDiskUse()
+        operation.isAllowDiskUse() == null
```

Comment:
Bad assertion?

Reply:
This is the find operation not the aggregation operation.